understanding/20/focus-order.html

```diff
@@ -54,6 +54,28 @@ <h2>Intent of Focus Order</h2>
          Web page. Care should be taken so that the focus order makes sense to both of these
          sets of users and does not appear to either of them to jump around randomly. 
       </p>
+
+      <p>Focus order also does not need to follow the visual presentation of the web page,
+         as long as the order in which elements receive focus is logical, and the hierarchy and relationship
+         of content implied by the visual presentation is preserved. For instance, in a two-column
+         layout, visually the assumption in western left-to-right systems may be that elements in the
+         left-hand column would receive focus first. However, if meaning and operation are not affected,
+         it is not a failure if elements in the right-hand column receive focus first.
+         Again, as a best practice, care should be taken to make sure the focus order does not
+         appear random and illogical.
+      </p>
+
+      <div class="note">
+         <p>It is not a failure of this success criterion if static/non-interactive content
+         receives focus. Making an element that does not act as an actionable user interface component
+         focusable, or programmatically moving focus to such an element, does not necessarily constitute
+         a failure of this success criterion. This situation is only failures if these additional focus stops
+         significantly impede the operation of the content, or create illogical focus orders - for example, a control
```

Comment:
I don't think "significantly impede the operation of the content" is really covered or implied by the SC text, can we remove that and just have it as "create an illogical focus order"?

Reply:
Changed to @mbgower's suggestion